test/metabase/query_processor/middleware/annotate_test.clj

```diff
@@ -319,7 +319,7 @@
    :display_name "sum of Price + 1"}
   (qp.test-util/with-everything-store
     (data/$ids venues
-      (col-info-for-aggregation-clause [:sum [:+ $price 1]]))))
+               (col-info-for-aggregation-clause [:sum [:+ $price 1]]))))
```

Comment:
`data/$ids` should have `{:style/indent 1}` metadata, is Emacs indenting it this way for you? maybe it's busted